biome.jsonc

```diff
@@ -1,12 +1,20 @@
 {
+	"vcs": {
+		"enabled": true,
+		"clientKind": "git",
+		"useIgnoreFile": true
+	},
 	"files": {
 		"includes": [
 			"**",
-			"!**/.nyc_output",
-			"!**/coverage",
 			"!**/schemas",
 			"!**/schemas.orig",
 			"!**/dummy.json",
+
+			/**
+			 * `npm version` updates `package.json` in such a way that `biome format` sometimes breaks CI.
+			 * @see https://github.com/seriousme/openapi-schema-validator/pull/195#issuecomment-3193470031
+			 */
 			"!**/package.json"
```

Comment:
I re-added it and written a comment to clarify why the match is needed.
To achieve this, I changed the file extension to `jsonc`, which should be fine since it’s one of the default formats resolved by `biome`.